keyboards/ashpil/modelm_usbc/rules.mk

```diff
@@ -0,0 +1,34 @@
+# MCU name
+MCU = at90usb1286
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLEe = no 		# Enable support for HD44780 based LCDs
+COMBO_ENABLE = yes          # For reset combo
```

Comment:
This should move to `keyboards/ashpil/modelm_usbc/keymaps/ashpil/rules.mk` to keep it enabled for your keymap but disabled for the default, which still doesn't compile. `COMBO_ENABLE` requires the keymap(s) to have combos configured in source. (This will matter less if we fix this feature in core.)

```suggestion
```

Reply:
If we do want a reset combo to exist for all keymaps(and we probably do as the model M isn't the easiest to disassemble), it seems easier to do this with a combo rather than bootmagic. So I think this is fine, I added the combo back to `modelm_usbc/keymaps/default`